recipes/mnist/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "mnist" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0cb9592f7397c1749cff7085c21724de79e7496dc136d43fb91788c72d57f926
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >= 2.7
+    - numpy
+
+test:
+  imports:
+    - mnist
+    - mnist.test
+
+about:
+  home: https://github.com/datapythonista/mnist
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

Comment:
Added issue ( https://github.com/datapythonista/mnist/issues/8 ) upstream to note how this can be included in the `sdist` in the future.

Reply:
should I get rid of the `license_file` line if I added the license to `MANIFEST.in` then?

Reply:
@jakirkham removed this line about the license, and the license file in the recipe. Please let me know if this is not what you meant.